bitinformation_pipeline/bitinformation_pipeline.py

```diff
@@ -49,10 +65,21 @@ def get_bitinformation(ds, dim=None, mask=None, label=None, overwrite=False):
     if overwrite:
         info_per_bit = {}
         for var in ds.data_vars:
-            # nbits = ds[var].dtype.itemsize * 8
             X = ds[var].values
             Main.X = X
-            info_per_bit[var] = jl.eval("get_bitinformation(X)")
+            if "mask" in kwargs:
+                raise ValueError(
+                    "bitinformation_pipeline does not wrap the mask argument. Mask your xr.Dataset with NaNs instead."
+                )
+            if "dim" in kwargs:
+                if isinstance(kwargs["dim"], str):
+                    kwargs["dim"] = ds[var].get_axis_num(kwargs["dim"]) + 1
+            if "masked_value" not in kwargs:
+                kwargs["masked_value"] = "NaN"
+            kwargs_str = " ," + ", ".join([f"{k}={v}" for k, v in kwargs.items()])
+            kwargs_str = kwargs_str.replace("True", "true").replace("False", "false")
+            logging.debug(f"get_bitinformation(X{kwargs_str})")
+            info_per_bit[var] = jl.eval(f"get_bitinformation(X{kwargs_str})")
```

Comment:
And here we could loop over `kwargs["dim"]`. Is that how you think about list support for `dim`?

Reply:
I would implement it at the beginning of `get_bitinformation`:
```python
def get_bitinformation(ds, dim):
    if isinstance(dim, list):
        for d in dim:
            get_bitinformation(ds, d)
        # somehow aggregate d results
```
but lets do this in a different PR. this is getting to large already